### ISSUE TITLE
Overlays - Refinement to UI

### ISSUE THREAD

**Issue by ajw2060** [Geektoolkit/DynaframeProIssues#34]
This relates to Build d2db213.

The management UI could be refined as follows.

1) The Overlays page currently performs two separate functions; being "overlay design" and "overlay selection".  I suggest this be simplified by limiting this page to just "overlay design" and renaming the page from "Overlays" to "Overlay Design".  The "overlay selection" should be moved to the "Home" page.  This "Home" page could then offer a switch to enable the overlay feature, and a drop down to select the desired overlay.  This makes it consistent with the Infobar controls on the home page.  The infobar controls offer the infobar on/off control as well as the infobar setting.  The infobar setting is similar to the "overlay selection", except the infobar setting is not user configurable (and therefore does not need an "Infobar Design" capability).  

See also Issue #35 (use of a save button) for further refinement  of the UI.

**Comment by Geektoolkit** [comment]
Good point. The actual plan for this is to move the overlay selection to the playlist page so that each playlist has an overlay that it uses.  That will remove that portion from the overlay designer (I like the idea of renaming it to that).  The reason I want to enable per playlist overlays is the idea that a playlist may have digital photography that a user may want to know focal lengths and other settings, it may have movie posters where they want to show 'now showing' and such, and then they may have art portraits where they want to show artist names and gallery info.  I could then have a default overlay that could be used for all if a user wants something simple or consistent.  When this portion of the feature gets in it should be more clear.

**Comment by Geektoolkit** [comment]
So this is solved a different way...the overlay page now has an 'edit' mode and when that is off, the overlay drop down is a select. There will be additional selection UI's added for overlay overrides as well.

**Comment by ajw2060** [comment]
Hi Joe,

 

Great to see you are still working on Dynaframe.  As you will have noticed, I have stopped paying to Patreon as I have decommissioned by installation. 

 

It sounds like you are getting a number of my issues closed which is great news.

 

Cheers for now.

 

Regards

Ashley Woods
